generator/typescript/dropbox_types.d.ts

```diff
@@ -6,6 +6,10 @@ declare module DropboxTypes {
     clientId?: string;
     // Select user is only used by team endpoints. It specifies which user the team access token should be acting as.
     selectUser?: string;
+    // Root path used to access namespaces different from home namespace (team folders etc)
```

Comment:
this shouldn't be needed, the change in tstemplate is enough.

Reply:
changes reverted